app/src/main/java/com/ahmettekin/WeatherApp/service/CityService.java

```diff
@@ -42,44 +41,62 @@ public static CityService getInstance() {
     }
 
     public void writeDataLocalDatabase(String cityName, View view) {
-        Gson gson = new GsonBuilder().setLenient().create();
-
-        retrofit = new Retrofit.Builder()
-                .baseUrl(BASE_URL)
-                .addCallAdapterFactory(RxJava2CallAdapterFactory.create())
-                .addConverterFactory(GsonConverterFactory.create(gson))
-                .build();
-
-        CityAPI cityAPI = retrofit.create(CityAPI.class);
-        Call<List<CityModel>> call = cityAPI.getData();
-        call.enqueue(new Callback<List<CityModel>>() {
-            @Override
-            public void onResponse(Call<List<CityModel>> call, Response<List<CityModel>> response) {
-                if (response.isSuccessful()) {
-                    List<CityModel> responseList = response.body();
-                    cityModels = new ArrayList<>(responseList);
-                    boolean sehirBulundu = false;
-
-                    for (CityModel cityModel : cityModels) {
-                        if (cityModel.name.matches(Character.toUpperCase(cityName.charAt(0)) + cityName.substring(1))) {
-                            LocalDataClass.getInstance().veriYaz(cityModel.name, cityModel.id, view.getContext());
-                            sehirBulundu = true;
-                            Toast.makeText(view.getContext(), cityModel.name + cityAddedText, Toast.LENGTH_SHORT).show();
-                            break;
+            Gson gson = new GsonBuilder().setLenient().create();
+            retrofit = new Retrofit.Builder()
+                    .baseUrl(BASE_URL)
+                    .addCallAdapterFactory(RxJava2CallAdapterFactory.create())
+                    .addConverterFactory(GsonConverterFactory.create(gson))
+                    .build();
+
+            CityAPI cityAPI = retrofit.create(CityAPI.class);
+            Call<List<CityModel>> call = cityAPI.getData();
+            call.enqueue(new Callback<List<CityModel>>() {
+                @Override
+                public void onResponse(Call<List<CityModel>> call, Response<List<CityModel>> response) {
+                    if (response.isSuccessful()) {
+                        List<CityModel> responseList = response.body();
+                        cityModels = new ArrayList<>(responseList);
+                        boolean sehirBulundu = false;
+                        String enteredCityName = UpperCaseWords(cityName);
```

Comment:
I detect that this code is problematic. According to the [Bad practice (BAD_PRACTICE)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#bad-practice-bad-practice), [Nm: Method names should start with a lower case letter (NM_METHOD_NAMING_CONVENTION)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#nm-method-names-should-start-with-a-lower-case-letter-nm-method-naming-convention).
Methods should be verbs, in mixed case with the first letter lowercase, with the first letter of each internal word capitalized.